Awesome! Tests pass (and it's super fast to run). I see one warning on OS X:

```
    * Running task 'C:build'... ./romanize.c:26:15: warning: implicit declaration of function 'atoi' is invalid in C99 [-Wimplicit-function-declaration]
                print_roman(atoi(argv[1]));
                            ^
1 warning generated.
```

You wanna throw a `#include<stdlib.h>` at the top of romanize.c to silence it?

--- a/languages/C/romanize.c
+++ b/languages/C/romanize.c
@@ -1,4 +1,5 @@
 #include<stdio.h>
+#include<stdlib.h>
 
 static const char* roman[]=        {"I","IV","V","IX","X","XL","L","XC","C","CD","D","CM","M"  };
 static const unsigned int arabic[]={  1,   4,  5,   9, 10,  40, 50,  90,100, 400,500, 900,1000 };

